03-Custom-Actions-In-Python/04-Custom-Updater.ipynb

```diff
@@ -0,0 +1,386 @@
+{
```

Comment:
<p>In the earlier sections, you instantiated the class inline. I think it would be more clear to use the same style in all the sections.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>